README.md

```diff
@@ -203,6 +203,76 @@ An array of behaviours, each having *one* of these properties:
 - **action.returnValue**
 - **action.throwError**
 
+If `stub.actions` is set or updated explicitly, at least one call to `stub.callFn(fn)`, `stub.callOriginal()`, `stub.returnWith(val)`, `stub.throwWith(err)`, `stub.callback(...)`, `stub.callbackAtIndex(...)`, `stub.resolveWith(val)`, `stub.rejectWith(val)`, `stub.withActions(actions)` or `stub.withActions()` must be made.
+
+### stub.withActions(actions) *or* stub.withActions()
+
+Configures this stub to use the specified array of actions. See `stub.actions` above for the syntax of an action.
+
+Example:
+
+```js
+var fn = simple.stub()
+         .withActions([{ returnValue: 'a' }, { returnValue: 'b' }])
+var returned1 = fn()
+var returned2 = fn()
+assert.equal(fn.callCount, 2)
+assert.equal(returned1, 'a')
+assert.equal(returned2, 'b')
+```
+
+`withActions` is, just like the others, chainable:
```

Comment:
In the interest of keeping documentation brief, let's just show the second example.